cypress.config.ts

```diff
@@ -1,9 +1,26 @@
 import { defineConfig } from "cypress";
+require("dotenv").config();
 
 export default defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
       // implement node event listeners here
     },
+    env: {
+      oneLoginSandboxUsername: process.env["one-login-sandbox-username"],
+      oneLoginSandboxPassword: process.env["one-login-sandbox-password"],
+      oneLoginApplicantEmail: process.env["one-login-applicant-email"],
+      oneLoginApplicantPassword: process.env["one-login-applicant-password"],
+      oneLoginAdminEmail: process.env["one-login-admin-email"],
+      oneLoginAdminPassword: process.env["one-login-admin-password"],
+      oneLoginSuperAdminEmail: process.env["one-login-super-admin-email"],
+      oneLoginSuperAdminPassword: process.env["one-login-super-admin-password"],
+    },
+    watchForFileChanges: !!+process.env["watch-for--file-changes"],
```

Comment:
might make for an easier dev experience to configure this via different commands in `package.json`? would it make sense to set this to true for `cy:open` and false for `cy:run`?